platform/src/main/java/org/hillview/table/columns/BaseArrayColumn.java

```diff
@@ -29,24 +27,24 @@
  * Adds a missing bit vector to BaseColumn (if missing values are allowed)
  */
 public abstract class BaseArrayColumn extends BaseColumn implements Serializable {
-    @Nullable
-    protected BitSet missing = null;
+    protected BitSet missing;
 
     public BaseArrayColumn(final ColumnDescription description, final int size) {
         super(description);
         if (size < 0)
             throw new InvalidParameterException("Size must be positive: " + size);
-        if (this.description.allowMissing && !this.description.kind.isObject())
-            this.missing = new BitSet(size);
+        this.missing = new BitSet(size);
     }
 
     @Override
     public boolean isMissing(final int rowIndex) {
-        return this.description.allowMissing && Converters.checkNull(this.missing).get(rowIndex);
+        // Ignore the analysis complaint. this.missing can not be null.
```

Comment:
Why is it complaining?
There is an official IDEA comment for this.
Same observation below

Reply:
Stale comment. Will remove.